julia/test/internals.jl

```diff
@@ -0,0 +1,811 @@
+# Store specific byte patterns in these variables for the fuzzer. These
+# values are taken verbatim from the C++ function FuzzTest1.
```

Comment:
these are handwritten tests? Should also be in `tests/`, though maybe under `tests/julia'.

Reply:
Again, artefact of adding the julia package in here. Might be best to keep the tests for that package along with it rather than duplicating them in here until we are releasing the Julia package from this repository?

Reply:
Certainly I don't want duplication of anything.. but moving them would be great. Don't want Julia to be the exception :)